tests/vmi_multus_test.go

```diff
@@ -856,6 +854,12 @@ var _ = Describe("[Serial]SRIOV", func() {
 			mac2, err := tests.GenerateRandomMac()
 			Expect(err).ToNot(HaveOccurred())
 
+			// start peer machines with sriov interfaces from the same resource pool
+			// manually configure IP/link on sriov interfaces because there is
+			// no DHCP server to serve the address to the guest
+			vmi1 := getSriovVmi([]string{"sriov-link-enabled"}, cloudInitNetworkDataWithStaticIPsByMac(mac1.String(), cidrA))
+			vmi2 := getSriovVmi([]string{"sriov-link-enabled"}, cloudInitNetworkDataWithStaticIPsByMac(mac2.String(), cidrB))
```

Comment:
for testing you should remove the skip please, unless you did it manually
x2

Reply:
What skip ?

Reply:
https://github.com/kubevirt/kubevirt/pull/4448/files#diff-ac233e621be1a1855fe741915d860d91484c7f061fc42beb213f4f9e0a756f86R882 

L 882
L 887

Reply:
Done, thanks!

Reply:
Can you add please those 2 issues that were mentioned on the skip to the fix list ?